### PR TITLE
Added docs and scripts to MANIFEST

### PR DESCRIPTION
**Pull request by fonnesbeck**
Fixes install problem where docs are not included in distribution.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,5 +1,6 @@
 recursive-include pymc3/examples/data *
 recursive-include source *
+recursive-include docs *
 include requirements.txt
 include *.md *.rst
-
+include scripts/*.sh
```
Comment (ferrine) [inline MANIFEST.in]: missing new line

File: setup.py [hunk 2]
```diff
@@ -48,22 +48,8 @@
...
-                    'pymc3.tuning',
-                    'pymc3.tests',
-                    'pymc3.variational',
-                    'pymc3.external',
-                    'pymc3.gp',
-                    'pymc3.plots',
-                    'docs',
-                    '.',
-          ],
+          packages=find_packages(),
```
Comment (twiecki) [inline setup.py:51]: We had this before and it didn't work properly. We should make sure that pip install from git works.

Comment (fonnesbeck) [reply setup.py:51]: It works using `pip install -e .`

**Comment by twiecki** [comment]
Let's give it a try.

**Pull request merged**